ansible/roles/cartodb.app/tasks/setup_fdw.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+ - name: Get commondata database name
+   command: psql -d carto_db_development -t -c "select database_name from users where username='{{ cartodb_common_data_username }}'"
+   become_user: postgres
+   register: cartodb_fdw_database_result
+   when: cartodb_common_data_local
+
+ - name: Get commondata user id
+   command: psql -d carto_db_development -t -c "select id from users where username='{{ cartodb_common_data_username }}'"
+   become_user: postgres
+   register: cartodb_common_data_user_id_result
+   when: cartodb_common_data_local
+
+ - name: Get commondata API key
+   command: psql -d carto_db_development -t -c "select api_key from users where username='{{ cartodb_common_data_username }}'"
+   become_user: postgres
+   register: cartodb_common_data_api_key_result
+   when: cartodb_common_data_local
+
+ - name: Set local variables needed for further tasks
+   set_fact:
+     cartodb_fdw_database: "{{ cartodb_fdw_database_result.stdout.strip() }}"
+     cartodb_common_data_user_id: "{{ cartodb_common_data_user_id_result.stdout.strip() }}"
+     cartodb_common_data_api_key: "{{ cartodb_common_data_api_key_result.stdout.strip() }}"
+   when: cartodb_common_data_local
+
+ - name: Add a foreign data wrapper user to the commondata database
```

Comment:
Could this be done with `postgresql_user_module` instead? Looks like they have support for encrypted passwords too.